osmoutils/accum/accum_helpers.go

```diff
@@ -38,26 +38,11 @@ func GetPosition(accum AccumulatorObject, name string) (Record, error) {
 
 // Gets total unclaimed rewards, including existing and newly accrued unclaimed rewards
 func GetTotalRewards(accum AccumulatorObject, position Record) sdk.DecCoins {
-	totalRewards := position.UnclaimedRewards
+	totalRewards := position.UnclaimedRewardsTotal
 
 	// TODO: add a check that accum.value is greater than position.InitAccumValue
-	accumulatorRewards := accum.value.Sub(position.InitAccumValue).MulDec(position.NumShares)
+	accumulatorRewards := accum.valuePerShare.Sub(position.AccumValuePerShare).MulDec(position.NumShares)
 	totalRewards = totalRewards.Add(accumulatorRewards...)
 
 	return totalRewards
 }
-
-// validateAccumulatorValue validates the provided accumulator.
-// All coins in custom accumulator value must be non-negative.
-// Custom accumulator value must be a superset of the old accumulator value.
-// Fails if any coin is negative. On success, returns nil.
-func validateAccumulatorValue(customAccumulatorValue, oldPositionAccumulatorValue sdk.DecCoins) error {
```

Comment:
Note: this was unused. I remember adding it but it ended up being incorrect to have this validation